docs/0002-atlas-translations-management.rst

```diff
@@ -0,0 +1,162 @@
+Atlas Translations Management Design
+####################################
+
+Date: 25 March 2024
+
+Status
+******
+Accepted
+
+Context
+*******
+
+Open edX microservices and micro-frontends use the processes outlined in `OEP-58`_ to pull the latest
+translations from the `openedx-translations repository`_.
+
+The main changes that `OEP-58`_ introduced to the Open edX project are the following:
+
+- Extract the English translation source entries and commit them into the `openedx-translations repository`_
+- The `GitHub Transifex app`_ will automatically commit the translations to the `openedx-translations repository`_
+- Provide a command that utilizes the `atlas`_ tool to pull translations in both development and web server image builds
+
+Microservices, micro-frontends, XBlocks and plugins use as few translation files as possible to makes it easier for Translators
+to locate and translate relevant resources.
+
+`edx-platform`_ translations strings used to be split between 7 files, but this has been deprecated in favor of
+``edx-platform.po`` and ``edx-platform-js.po`` files.
+More details are available in the `edx-platform resources combination decision document`_.
+
+The `openedx-app-ios`_ uses a clean architecture. Therefore, each clean architecture module has its own i18n
+localizable strings file. This is useful for Developer Experience.
+However, it's not good for Translator Experience because translators
+often are unaware of the application engineering architecture and would like to be able to quickly search for strings
+regardless of their location in the code.
+
+Decision
+********
+
+This decision requires a few changes to the mobile app translation workflow:
+
+- Translation files are no longer comitted directly to the mobie app repositories (`openedx-app-ios`_ / `openedx-app-android`_)
+- Keep the i18n string files split into each module without hurting Developer Experience.
+- String files are combined into a single file before uploading it to Transifex
+  (through the `openedx-translations repository`_)
+- Translators can translate the entire app within a single transifex resource
+- Before releasing to App Store, developers need to run a new ``make pull_translations`` command
+- After translations are pulled the translation files are split into each module
+  as if it has been individual resource (a new script)
```

Comment:
This section is a little confusing. Some bullet points are describing changes to the workflow, some are describing ways in which the workflow will not change, and some are just describing the proposed new workflow without reference to anything changing/staying the same.

It might be helpful to split this up into a couple subsections:
```
New Workflow
* Bullets
* Describing
* New
* Workflow
```
```
Notable changes
* Bullets describing notable ways in which the new workflow is different from the previous workflow
```

Or maybe it could all be done in one section focusing on the new workflow, with sub-bullets under any bullets that include notable changes.